packages/react-dom/src/server/ReactPartialRendererContext.js

```diff
@@ -77,16 +77,20 @@ export function processContext(
   const contextType = type.contextType;
   if (typeof contextType === 'object' && contextType !== null) {
     if (__DEV__) {
-      if (contextType.$$typeof !== REACT_CONTEXT_TYPE) {
+      const isContextConsumer =
+        contextType.$$typeof === REACT_CONTEXT_TYPE &&
+        contextType._context !== undefined;
```

Comment:
Was it also called `context` in some patch release?

Reply:
Yeah, it was changed to `_context` in https://github.com/facebook/react/pull/12501. Is it worth checking? I think a lot more would be broken if you were trying to render a tree created with a version of React that still used `context`